data/transport missions.txt

```diff
@@ -1951,3 +1951,244 @@ ship "Science Drone"
+
+
+
+mission "Pirate Blockade"
+	minor
+	name `Escape to Freedom`
+	description `Bring <bunks> escaped slaves to freedom from their pirate warlords on <destination>. The woman on <origin> promised that a person on <planet> would give you <payment>.`
+	source "Buccaneer Bay"
+	destination "Hippocrates"
+	passengers 19
+	to offer
+		"combat rating" > 500
+	
+	on offer
+		conversation
+			`It is dusk at the <origin> spaceport, and the setting sun casts long shadows across the spaceport deck. You take notice of a slim cloaked figure walking only a few feet behind you who seems to be keeping pace.`
+			choice
+				`	(Confront them.)`
+					goto confront
+				`	(Stop walking and see if they stop as well.)`
+			
+			`	You stop be the entrance to a store that claims to have "the best Polecats!" and lean against the building, trying not to draw attention to yourself. The figure keeps walking at the same pace, and as they pass you, a soft voice says, "Follow me," from under the cloak. The figure bolts off into a dark alleyway next to the building.`
+				goto choice
+			
+			label confront
+			`	You turn around and stare right at the figure, who stops within a foot of you. The shadows cast by the sun make it impossible for you to make out their face under the cloak. "Follow me," a soft voice says from under the cloak, before the figure bolts off into a dark alleyway next to the building.`
+			
+			label choice
+			choice
+				`	(Follow the figure.)`
+				`	(Return to my ship.)`
+					decline
+			
+			`	You step into the alleyway and find the figure standing under a lamp. The figure removes their cloak, revealing the face of a woman with short dark hair and a distinct jagged scar across her right cheek. She begins whispering to you. "You're not from this world. I can tell. You're too clean and unscarred. Would you be able to help me?"`
+			choice
+				`	"What do you want?"`
+					goto need
+				`	"Why would I help a pirate?"`
+			
+			`	The woman chuckles. "Not everyone who lives on a pirate world is a pirate, you know. Maybe you need to learn how not to stereotype. Regardless..." She trails off.`
+			
+			label need
+			`	"It isn't what I want, it's what I need," she whispers, putting heavy emphasis on the word "need."`
+			`	"You undoubtedly know about the horrendous slave trade that still exists on pirate worlds today. A slave trade that the Republic doesn't care to abolish, leaving innocent people to waste away like rats in a cage, working for their lives. As if the politicians of the Republic aren't vermin themselves..." Her face twists into a hate filled scorn as she speaks of the Republic. It then returns to normal as she continues. "I'm part of a group of people who have decided to take matters into our own hands. We're mostly people who grew up on pirate worlds, but some of us were born elsewhere."`
```

Comment:
abolish -> do anything about [Presumably slavery is already illegal, so it's not that the Republic hasn't abolished it, but rather, they're not enforcing the abolition]

working for their lives. -> [working for their whole lives, toiling for their whole lives, working without any respite, etc.]

a hate filled scorn -> a look of hate-filled scorn

Reply:
>abolish  -> do anything about [Presumably slavery is already illegal, so it's not that the Republic hasn't abolished it, but rather, they're not enforcing the abolition]

Abolish just seems like a stronger word better fit for the scenario. And from her perspective, slavery hasn't been abolished, even if there are laws in the Republic that have abolished slavery. That's exactly why she tries to free slaves; she IS trying to abolish the slave trade.

>working for their lives. -> [working for their whole lives, toiling for their whole lives, working without any respite, etc.]

The implication with those words is that if they don't work, they get killed.

Reply:
> Abolish just seems like a stronger word better fit for the scenario. 

Eh, fair enough.

> The implication with those words is that if they don't work, they get killed.

Might wanna make that a bit clearer.

Reply:
I might just get clearer about that... in future missions. :)